roles/openshift_efk/tasks/main.yaml

```diff
@@ -0,0 +1,179 @@
+---
+
+  - fail: msg="This role requires the following vars to be defined: master_url, kibana_hostname, es_cluster_size"
+    when: "kibana_hostname is not defined or
+          es_cluster_size is not defined or
+          master_url is not defined"
+
+  - name: "Checking for logging project"
+    command: oc get project logging
+    register: logging_project
+    failed_when: "'FAILED' in logging_project.stderr"
+    tags:
+      - cleanup
```

Comment:
Also can you explain to me what this line is actually saying:
`not openshift_hosted_logging_cleanup | default(false) | bool`

Im confused as to where bool coming from?